build.gradle

```diff
@@ -22,49 +28,79 @@ dependencies {
 }
 
 task sourcesJar(type: Jar, dependsOn: classes) {
-    classifier = 'sources'
+    archiveClassifier.set('sources')
     from sourceSets.main.allSource
 }
 
 task javadocJar(type: Jar, dependsOn: javadoc) {
-    classifier = 'javadoc'
+    archiveClassifier.set('javadoc')
     from javadoc.destinationDir
 }
 
-task createPom {
-    doLast {
-        pom {
-            project {
-                groupId 'io.tus.java.client'
-                artifactId 'tus-java-client'
-                name 'tus-java-client'
-                version project.getProperties().get('pomVersion')
-                packaging 'pom'
-                description 'Java client for tus, the resumable file uploading protocol.'
-                url 'http://tus.io'
-
-                scm {
-                    url 'https://github.com/tus/tus-java-client'
-                    connection 'https://github.com/tus/tus-java-client'
-                    developerConnection 'https://github.com/tus/tus-java-client'
-                }
-
-                developers {
-                    developer {
-                        id 'acconut'
-                        name 'Marius Kleidl'
-                        email 'maerious@gmail.com'
-                    }
-                }
-
-                inceptionYear '2015'
-                licenses {
-                    license {
-                        name 'The MIT License (MIT)'
-                        url 'http://opensource.org/licenses/MIT'
-                    }
-                }
+artifacts {
+    archives sourcesJar, javadocJar
+}
+
+def versionNo = rootProject.file('version.txt').text.trim()
```

Comment:
Can we use the same version.properties file as in the Java SDK? i.e. https://github.com/transloadit/java-sdk/blob/807e997e48c1697cf557bfb1d97a612ac62e7866/build.gradle#L14, to bring the two scripts closer together

Reply:
Yes